addons/l10n_de_skr03/data/account.account.template.csv

```diff
@@ -0,0 +1,1283 @@
+"id","code","name","tag_ids/id","user_type_id/id","chart_template_id/id","reconcile"
+"account_0005","0005","Rückständige fällige Einzahlungen auf Geschäftsanteile","l10n_de.tag_de_asset_bs_A_I_2","account.data_account_type_non_current_assets","l10n_de_skr03.l10n_de_chart_template"
+"account_0010","0010","Entgeltlich erworbene Konzessionen, gewerbliche Schutzrechte und ähnliche Rechte und Werte sowie Lizenzen an solchen Rechten und Werten","l10n_de.tag_de_asset_bs_A_I_2","account.data_account_type_non_current_assets","l10n_de_skr03.l10n_de_chart_template"
```

Comment:
Because it's a csv, I prefer the value of ``reconcile`` is always set even in case of ``False``

Reply:
changes done.